### DISCUSSION TITLE
Needing some help family on training a SD3 on like 30 images or even just determine if its to much its for characters and people

### DISCUSSION THREAD

**Discussion by TheMindExpansionNetwork** [bghira/SimpleTuner#702] [Q&A]
Hello, all trying to find the sweet spot for some SD3 training got the SD2 down, but you know we need it more powerful.

I am just wanting to know all your settings to determine what is best because at 30 photos settings take like 18 hours but I am not sure that is just what it is going to take now with the new aspects of what we are building you know its just more power to be able to play.

Please let me know your experiences if I am just going to have to bite the bullet for overnight fun or if there is anything else to better speed up things.

If able please drop your config for something good.

thank you all for your help and much love

**Comment by bghira** [comment]
i think your config is going to do like 1000 epochs and you'll probably be good before 18 hours is up

**Reply by TheMindExpansionNetwork** [reply]
Ok I see so I just keep evaluating until I got the sweet spot

**Reply by bghira** [reply]
yeah for a single subject maybe 1000 steps should be good for a real person and like 5000-10000 for animations it doesn't know as a guess
